src/components/link/props.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+// This file exports an array of all props that are valid on an `a` (anchor)
```

Comment:
Don't we need to whitelist `role` as well?

Reply:
Tbh, I copied that from another codebase I work on.
It’s probably missing a lot of attributes, including `role`, indeed. Probably a large number of `aria-…` attributes too. 

IMO this is the most foolproof solution, but it would be much better if a module handled that for us, maintaining it is a pain.

Two modules exist but aren’t so popular:
- https://www.npmjs.com/package/filter-react-props (it seems to have a pretty complete list: https://github.com/asistapl/filter-react-props/blob/master/src/index.js)
- https://badge.fury.io/js/filter-react-dom-props 

I wish this was exposed by `react-dom` instead!

Reply:
But yeah, I should probably add `role` to the list. Ideally you’d add new attributes whenever you’re using one that isn’t whitelisted, but that’s annoying.

I’m thinking of actually not doing that at all and simply filtering out `isHidden` and `loggedOut` like it’s done for the `react-router` props, and leave a comment explaining what these are.

Reply:
We actually maintain that list for styled-components! For now it should be good enough to specifically filter out those props though, that looks fine to me.

Reply:
So, you’re fine with how it is right now? :)